src/app/course-page/course-page.component.html

```diff
@@ -1,7 +1,19 @@
 <app-header></app-header>
-<hr>
-<p>course-page works!</p>
-<course-search></course-search>
-<course-lists></course-lists>
-<hr>
-<app-footer></app-footer>
\ No newline at end of file
+<div class="course-page-main">
+  <div class="course-page-title-container">
+    <h4 class="h4 course-page-title">Courses</h4>
+  </div>
+  <div class="course-page-container">
+    <div class="course-page-search">
+      <course-search
+        (emitSearchElement)="searchElement($event)"
```

Comment:
usually, event don't prefix with `emit`
It's more common to see this pattern when your event represents what it doest and the handler is prefixed with `on` `(search)="onSearch($event)`
you can also add `course` to the name to be more specific)
`(search)="onCourseSearch($event)`

Reply:
done